readme.md

```diff
@@ -65,6 +65,15 @@ updates smaller.
 There is also a Python Wheel provided of the application which can be installed into
 a virtual environment for those familiar with Python virtual environments.
 
+### From Source ###
+
+If you're on Windows and have [uv](https://docs.astral.sh/uv/),
+you should be able to navigate to the base project folder and launch with:
+
+```
+uv run python -m splitguides
+```
+
```

Comment:
Can you leave this readme edit out.

Building from source should only really be something anyone does if they wish to do work on development and isn't something I want to suggest to end users. There are further instructions developers would require (for example, they need to install the test dependencies included in the `testing` extra or the `build` extra for creating a windows executable).